tests/api/v1/aspects/patch.js

```diff
@@ -174,6 +174,240 @@ describe('tests/api/v1/aspects/patch.js >', () => {
         return done();
       });
     });
+
+    describe(`PATCH ${path} helpEmail, helpUrl not set in db >`, () => {
+      before(() => tu.toggleOverride('requireHelpEmailOrHelpUrl', true));
+      after(() => tu.toggleOverride('requireHelpEmailOrHelpUrl', false));
```

Comment:
the "after" function should be restoring the toggle to its original state, and the original state is not necessarily false -- better to save the original value in the before, before doing the override